google-oauth-client/src/test/java/com/google/api/client/auth/oauth/OAuthHmacSignerTest.java

```diff
@@ -32,4 +35,39 @@ public void testComputeSignature() throws GeneralSecurityException {
     signer.tokenSharedSecret = "def";
     assertEquals(EXPECTED_SIGNATURE, signer.computeSignature("foo"));
   }
+
+  public void testComputeSignatureHmacSha256() throws GeneralSecurityException {
+    final OAuthHmacSigner signer = new OAuthHmacSigner();
+    signer.setSignatureMethod("HMAC-SHA256");
+    signer.clientSharedSecret = "apiSecret";
+    signer.tokenSharedSecret = "tokenSecret";
+    final String expected = "xDJIQbKJTwGumZFvSG1V3ctym2tz6kD8fKGWPr5ImPU=";
+    assertEquals(expected, signer.computeSignature("baseString"));
+  }
+
+  public void testGetSignatureMethod() {
+    final OAuthHmacSigner signer = new OAuthHmacSigner();
+    final String expected = "HMAC-SHA1";
+    assertEquals(expected, signer.getSignatureMethod());
+  }
+
+  public void testGetSignatureMethodHmacSha256() {
+    final OAuthHmacSigner signer = new OAuthHmacSigner();
+    final String signatureMethod = "HMAC-SHA256";
+    signer.setSignatureMethod(signatureMethod);
+    assertEquals(signatureMethod, signer.getSignatureMethod());
+  }
+
+  public void testSetSignatureMethodHmacMD5() {
+    assertThrows(
+            "Signature method HMAC-MD5 not available",
```

Comment:
four space indents for continuation lines, per Google style guide

Reply:
this has been reverted back in favor of a new class